ssb/src/main/java/com/google/cloud/CloudMonitoringResult.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud;
```

Comment:
we probably don't want these class in the cloud package. something like `com.google.cloud.storage.benchmarking` where it's still under storage, but not in the storage package itself so that it will always use the public api.

Reply:
this is probably a naive question but can I just rename the package or do I need to do more to get it structurally under `com.google.cloud.storage.benchmarking`

Reply:
Intellij should be able to move everything for you by running the move refactoring.

In the project files window select all of the classes, then run `Refactor > Move...`
![image](https://github.com/googleapis/java-storage/assets/1754037/513fbcb7-b16a-48fb-bc9b-f9e847618acb)

Type in the new package, and IntelliJ will update everything, including the main class in the POM.
![image](https://github.com/googleapis/java-storage/assets/1754037/7fb00414-c113-49fb-9e22-20c4e053d0be)